instrumentation/google.golang.org/grpc/otelgrpc/interceptor.go

```diff
@@ -233,6 +233,15 @@ func (w *clientStream) sendStreamEvent(eventType streamEventType, err error) {
 	}
 }
 
+func target(cc *grpc.ClientConn) string {
```

Comment:
This doesn't require a method (which won't be used for anything else). We could be setting a variable.